content/blog/2020/10/Camel36-Whatsnew/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Apache Camel 3.6 What's New"
+date: 2020-10-21
+authors: [davsclaus]
```

Comment:
I think quotes are required, I could be wrong
```suggestion
authors: ["davsclaus"]
```

Reply:
Its a copy from 3.5 blog post that was published